tests/bulkdata/jsonld_import_tests.py

```diff
@@ -149,7 +149,7 @@ def test_write(self):
             """
             ),
         )
-        importer = JSONLDImporter(request=request, loadid=str(start_event.pk))
+        importer = JSONLDImporter(request=request)
```

Comment:
This captures the original failure -- revert the change and keep this edit to the test to produce the failure. Apparently the importers are only instantiated with a request kwarg.

Reply:
I guess it meant to one OR the other probably not both.